src/tests/sharedUtils.tsx

```diff
@@ -65,6 +69,21 @@ export const DEFAULT_REFERENTIELIJSTEN_TABELLEN: ReferentielijstenTabelOption[]
   },
 ];
 
+export const DEFAULT_REFERENTIELIJSTEN_TABEL_ITEMS: Record<string, ReferentielijstenTabelItem[]> = {
+  countries: [
+    {
+      code: 'option1',
+      naam: 'Option 1',
+      isGeldig: true,
+    },
+    {
+      code: 'option2',
+      naam: 'Option 2',
+      isGeldig: true,
+    },
+  ],
```

Comment:
Let's use some real data :)

```suggestion
  countries: [
    {
      code: 'nl',
      naam: 'The Netherlands',
      isGeldig: true,
    },
    {
      code: 'gr',
      naam: 'Greece',
      isGeldig: true,
    },
  ],
```